parsers/thueringen.py

```diff
@@ -0,0 +1,169 @@
+from urllib.request import urlopen
+from bs4 import BeautifulSoup as parse
+import re
+from utils import Parser
+from datetime import datetime, timedelta, date
+from pyopenmensa.feed import LazyBuilder
+
+amount_regex = re.compile('\d{1,2},\d{1,2}')
+
+
+def parse_start_date(document):
+	week_start_end_date_regex = re.compile('\d{1,2}\.\d{1,2}\.\d{4}')
+	calendar_week_regex = re.compile('\d{4}-\d{1,2}$')
+
+	option = document.find('option', value=calendar_week_regex, selected=True)
+
+	start_date_str = week_start_end_date_regex.search(option.text)
+
+	return datetime.strptime(start_date_str.group(), '%d.%m.%Y').date()
+
+
+def parse_available_weeks(document):
+
```

Comment:
NIT: I find empty lines after function definitions of for loops (see below) a little bit confusing. They also seem to confuse the git diff algorithm. :wink: